src/bin/pgcopydb/schema.c

```diff
@@ -4769,9 +4777,28 @@ getTableArray(void *ctx, PGresult *result)
 
 		if (context->catalog != NULL && context->catalog->db != NULL)
 		{
-			if (!catalog_add_s_table(context->catalog, table))
+			if (table->relkind == 'r' || table->relkind == 'p')
 			{
```

Comment:
Nitpicking: can we use a `swich(table->relkind)` statement? I find those easier to follow when maintaining the code.

Reply:
Fixed it. Thank you.